components/list_modal.tsx

```diff
@@ -1,87 +1,46 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import PropTypes from 'prop-types';
 import React from 'react';
 import {Modal} from 'react-bootstrap';
 import {FormattedMessage} from 'react-intl';
 
 import LoadingScreen from 'components/loading_screen';
+import {Group} from '@mattermost/types/groups';
 
 export const DEFAULT_NUM_PER_PAGE = 50;
 
-export default class ListModal extends React.PureComponent {
-    static propTypes = {
-
-        /**
```

Comment:
As I was informed previously, these comments are picked up by some tool for documentation. For this reason, I would keep the comments

Reply:
Thanks for adding back the comments